src/app/pages/feedback/feedback.page.ts

```diff
@@ -1,54 +1,71 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Router } from '@angular/router';
+import { ModalController, NavParams } from '@ionic/angular';
 import * as _ from 'lodash-es';
+import { localKeys } from 'src/app/core/constants/localStorage.keys';
+import { LocalStorageService, ToastService } from 'src/app/core/services';
+import { SessionService } from 'src/app/core/services/session/session.service';
+import { DynamicFormComponent, JsonFormData } from 'src/app/shared/components/dynamic-form/dynamic-form.component';
 
 @Component({
   selector: 'app-feedback',
   templateUrl: './feedback.page.html',
   styleUrls: ['./feedback.page.scss'],
 })
 export class FeedbackPage implements OnInit {
-  public headerConfig: any = {
-    // menu: true,
-    backButton: {
-      label: '',
-    },
-    notification: false,
-    headerColor: 'white',
-  };
-  formData: any=[];
-  constructor() {}
+  @ViewChild('form1') form1: DynamicFormComponent;
 
-  ngOnInit() {}
+  formData: JsonFormData = {
+    controls: [],
+  };
+  feedbackData = {
+    feedbacks: [],
+  };
+  sessionData: any;
+  isMentor: boolean;
+  constructor(private sessionService: SessionService,
+    private toast: ToastService,
+    private router: Router,
+    private modalController: ModalController,
+    private navParams: NavParams,
+    private localStorage: LocalStorageService) {
+    this.sessionData = this.navParams?.data?.data;
+  }
 
-  ionViewWillEnter() {
-    this.formData = [
-      {
-        label: 'How would you rate the host of the session?',
-        numberOfStars:5,
-        value: 0,
-      },
-      {
-        label: 'How would you rate the engagement of the session?',
-        numberOfStars:6,
-        value: 0,
-      },
-      {
-        label: 'How would you rate the Audio/Vedio quality?',
-        numberOfStars:7,
-        value: 0,
-      },
-    ];
+  ngOnInit() {
+    this.isMentorChecking();
+  }
+  async isMentorChecking() {
+    let userDetails = await this.localStorage.getLocalData(localKeys.USER_DETAILS);
+    var response = await this.sessionService.getSessionDetailsAPI(this.sessionData._id);
+    this.isMentor = userDetails._id == response.userId ? true : false;
```

Comment:
why do we need this check

Reply:
For showing the corresponding feedback heading. for both mentor and mentee